### PR TITLE
[mypy] fix small folders

### PR DESCRIPTION
**Pull request by algobytewise**
### **Describe your change:**
Related Issue: #4052

comment for strassen_matrix_multiplication.py: I couldn't use Tuple as return type since one branch of the function returns a tuple an another a list/matrix. So I turned the tuple into a list instead.

comment for electric_power.py & ohms_law.py: I added a final else-clause for raising an error to make sure that each branch either has a return-value or throws an error

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -29,12 +29,16 @@ jobs:
...
+           electronics
+           file_transfer
            fractals
            fuzzy_logic
            genetic_algorithm
            geodesy
            knapsack
            networking_flow
+           quantum
            scheduling sorts
```
Comment (cclauss) [inline .github/workflows/build.yml:42]:
```suggestion
           scheduling
           sorts
```

File: file_transfer/receive_file.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
         print("Receiving data...")
         while True:
             data = sock.recv(1024)
-            print(f"data={data}")
+            print(f"data={str(data)}")
```
Comment (cclauss) [inline file_transfer/receive_file.py:16]: Why this change?

File: file_transfer/send_file.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ def send_file(filename: str = "mytext.txt", testing: bool = False) -> None:
         conn, addr = sock.accept()  # Establish connection with client.
         print(f"Got connection from {addr}")
         data = conn.recv(1024)
-        print(f"Server received {data}")
+        print(f"Server received {str(data)}")
```
Comment (cclauss) [inline file_transfer/send_file.py:16]: Why this change?

File: divide_and_conquer/max_difference_pair.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
-from typing import List
+from typing import List, Tuple
```
Comment (cclauss) [inline divide_and_conquer/max_difference_pair.py:1]: @dhruvmanila Do we need these imports for basic types in Python 3.9 and the current mypy?

File: file_transfer/receive_file.py (hunk 2 above)
Comment (dhruvmanila) [reply file_transfer/receive_file.py:16]: This and similar changes are redundant as string interpolation converts the object to string using the `__str__` method of the object.

File: divide_and_conquer/max_difference_pair.py (hunk 4 above)
Comment (dhruvmanila) [reply divide_and_conquer/max_difference_pair.py:1]: No, I updated the issue with the message: https://github.com/TheAlgorithms/Python/issues/4052#issuecomment-783919171

File: file_transfer/receive_file.py (hunk 2 above)
Comment (algobytewise) [reply file_transfer/receive_file.py]: I'm not sure why but without it we get the following mypy-errors:
file_transfer/send_file.py:16: error: On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior
file_transfer/receive_file.py:16: error: On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior

File: divide_and_conquer/max_difference_pair.py (hunk 4 above)
Comment (algobytewise) [reply divide_and_conquer/max_difference_pair.py]: I see, I made the corresponding changes. One difference is that the basic types are spelled lower case with this import

File: file_transfer/receive_file.py (hunk 2 above)
Comment (algobytewise) [reply file_transfer/receive_file.py:16]: I guess it has something to do with the method from the socket-package that returns the data-object.

Comment (cclauss) [reply file_transfer/receive_file.py:16]: `bytes` vs. `str` is the most time-consuming part of porting code from Python 2 to Python 3.
% __python3__
```
>>> type(b"abc")
<class 'bytes'>
>>> type("abc")
<class 'str'>
```

File: file_transfer/send_file.py (hunk 3 above)
Comment (cclauss) [inline file_transfer/send_file.py:16]:
```suggestion
        print(f"Server received: {data = }")
```

**Review by cclauss — APPROVED** [review]
LGTM...
